README.md

```diff
@@ -114,7 +114,7 @@ The syslog facility that should be used for logging purposes.
 
 pid_file
 --------
-File in which the NRPE daemon should write it's process ID number.
+File in which the NRPE daemon should write it's process ID number. To disable pid_file, specify 'nrpe::pid_file: absent' in hiera.
```

Comment:
Hiera isn't actually necessary, which might confuse people.

`To disable the use of the `pid_file` parameter, specify the value as 'absent'.`